toolkits/global/packages/global-forms/README.md

```diff
@@ -140,216 +217,258 @@ You can summarise errors using a top level `errorSummary` property. Each error i
+Unlike radios, which are always used in sets of 2 or more, you can have a single checkbox field. To give users a set of checkbox choices, organise the checkboxes into a `fieldset`:
 
-```json
-"svg": "<svg xmlns=\"http://www.w3.org/2000/svg\" style=\"--forms--pictographic-radio-icon-size: 2rem\" focusable=\"false\" aria-hidden=\"true\" viewBox=\"0 0 24 24\">...</svg>"
 ```
-
-#### Checkboxes
-
-Unlike radios, which are always used in sets of two or more, you can have a single checkbox field. To give users a set of checkbox choices, organise the checkboxes into a `fieldset`:
-
-```json
 {
-    "legend": "<h2>Which animals do you like?</h2>",
+    "legend": "<h2>Which subject categories apply to your manuscript?</h2>",
     "fields": [
         {
             "template": "globalFormCheckbox",
-            "label": "Monkeys",
-            "id": "checkbox-monkeys",
-            "name": "checkbox-monkeys"
+            "label": "Biological sciences",
+            "id": "biological-sciences",
+            "name": "biological-sciences"
+        },
+        {
+            "template": "globalFormCheckbox",
+            "label": "Earth and environmental sciences",
+            "id": "earth-sciences",
+            "name": "earth-sciences"            
+        },
+        {
+            "template": "globalFormCheckbox",
+            "label": "Health sciences",
+            "id": "health-sciences",
+            "name": "health-sciences"            
         },
         {
             "template": "globalFormCheckbox",
-            "label": "Horses",
-            "id": "checkbox-horses",
-            "name": "checkbox-horses"
+            "label": "Physical sciences",
+            "id": "physical-sciences",
+            "name": "physical-sciences"            
         }
     ]
 }
 ```
 
-#### Hidden fields
 
-You can hide any field from visibility (also from screen reader software and keyboard interaction) using `hidden: true`. If you want to use a field of `type="hidden"`, you need to use `template: globalFormHidden`. It is recommended you also apply `hidden: true` to such fields because it will remove the inter-field margin. In the following example, note the `label` is omitted since hidden fields are not user facing.
+## Selects
 
-```json
- {
-    "template": "globalFormHidden",
-    "id": "hidden-field",
-    "name": "hidden-field",
-    "hidden": true,
-}
-```
+![Selects](https://user-images.githubusercontent.com/15365576/210582769-2eb6fc0e-ce10-40f4-b6a0-d037e5c55c0a.png)
 
-#### Inline fields
+### When to use this form element
 
-It is possible to display fields in a horizontal line, using the following `template` value:
+Research shows that some [people find selects difficult to use](https://www.lukew.com/ff/entry.asp?1950).
 
-```json
-"template": "globalFormInlineFields"
-```
+Only use selects when there’s genuinely no other option.
```

Comment:
I think this is redundant (and a bit vague) the following text covers alternatives nicely.